### DISCUSSION TITLE
Problem installing

### DISCUSSION THREAD

**Discussion by M4CH1N3BR** [AAFOptimus/AAFFamilyDCHAudio#1] [Q&A] [answered]
Hello friend, thank you for continuing with the project. I was using version AAFOptimusAudioPack_9655 perfectly, so I decided to format my PC and install Windows 11 Pro, 24H2, OS build 26100.2314, and after that I can no longer install AAFOptimus. Do you have any idea what might have happened?

Note (The driver goes through the installation process but does not install any APP and the audio driver remains the same as Microsoft's, but when I open the control panel and look to uninstall programs, AAF Optimus is there)

My motherboard is a Gigabyte Z270xp, my audio card is
ALC1220 120dB SNR HD Audio

HDAUDIO\FUNC_01&VEN_10EC&DEV_0900&SUBSYS_1458A182&REV_1000

Note. I installed an old version AAFOptimusAudioPack9231 and it worked perfectly.

**Comment by AAFOptimus** [comment]
Hello.
Thanks for the feedback.
[Please try the latest version](https://github.com/AAFOptimus/AAFFamilyDCHAudio/releases/latest)

**Reply by M4CH1N3BR** [reply]
Thanks for replying, Yes, I have already tried the latest version posted by you and several others that I found on the internet, the process of each installation is very clean using Driver Storage Explorer and removing all residues, to be honest I have already tried to install it about 15 times.

**Comment by M4CH1N3BR** [comment]
Maybe something wrong with this version of windows?

![image](https://github.com/user-attachments/assets/e11c45a3-ec3b-4a04-a488-c630f95fa132)

**Reply by ragavendranbala-TSL** [reply, marked as answer]
24H2 having some issues in audio driver. This mod currently doesn't support Windows 11 24H2. But regarding no installation of apps, kindly try the new release available on git but 24H2 is not supported at the moment

**Reply by M4CH1N3BR** [reply]
It was what I suspected, I spent hours searching the Microsoft website for an answer, many people had the same problem, thanks for the answer.
